.github/workflows/updateReadme.yml

```diff
@@ -0,0 +1,53 @@
+name: 'update-readme'
+
+on:
+  # cron schedule runs every Tuesday at 5pm UTC following public release of new cloud sdk version
+  schedule:
+    -cron: '0 17 * * 2'
+
+jobs:
+  build:
+    permissions:
+      contents: 'write'
+      id-token: 'write'
+      pull-requests: write
+    runs-on: 'ubuntu-latest'
+
+    steps:
+    - uses: 'actions/checkout@v3'
+
+    - uses: 'actions/setup-node@v3'
+      with:
+        node-version: '16.x'
+    
+    # Used to authenticate setup-gcloud in update script
+    - name: 'Auth with WIF'
+      uses: google-github-actions/auth@v0
+      with:
+        workload_identity_provider: # TODO replace with WIF
+        service_account: # TODO: replace with SA
```

Comment:
Do we need auth for this? `docker run -ti  google/cloud-sdk:latest gcloud functions event-types list` seems to work without auth

Reply:
Had not realized auth wouldn't be required (or rather had not tested without). Can omit the auth step and just run the command with a setup gcloud step prior.